lib/decimal_to_binary.rb

```diff
@@ -0,0 +1,20 @@
+
+def decimal_to_binary(decimal)
```

Comment:
👍 Nice work

You can use `bin = decimal % 2` to get the remainder of the decimal number divided by 2.